github/repos.go

```diff
@@ -579,6 +582,10 @@ type PullRequestReviewsEnforcementRequest struct {
 	DismissStaleReviews bool `json:"dismiss_stale_reviews"`
 	// RequireCodeOwnerReviews specifies if an approved review is required in pull requests including files with a designated code owner.
 	RequireCodeOwnerReviews bool `json:"require_code_owner_reviews"`
+	// RequiredApprovingReviewCount specifies the number of approvals required before the pull request can be merged.
+	// Valid values are 1-6.
+	// Can be omitted.
+	RequiredApprovingReviewCount int `json:"required_approving_review_count,omitempty"`
```

Comment:
If it can be omitted, then I think we need to make this type `*int` instead of `int`.

Reply:
Issue with the comment, I forgot to update it. Its a required field.

Reply:
Then in that case, should we get rid of the `,omitempty` here and below?